packages/react-components/react-select/src/components/Select/useSelectStyles.ts

```diff
@@ -108,25 +148,45 @@ const useSelectStyles = makeStyles({
       ...shorthands.borderColor('GrayText'),
     },
   },
+
   small: {
     height: fieldHeights.small,
-    ...shorthands.padding('0', tokens.spacingHorizontalSNudge),
+    paddingBottom: 0,
```

Comment:
The `paddingBottom/Top` values are the same for all sizes. Can these be moved to the `base` styles?